Doc - 3.3 known issues page (#7575)
arangodb/arangodb: Documentation/Books/Manual/ReleaseNotes/KnownIssues33.md

Known Issues
============

> This page lists some known issues of the ArangoDB suite of products. 

Reviewer: Maybe make this text a bit more useful and clarify what this is and what it isn't?
```suggestion
This page lists important issues of the ArangoDB suite of products all users
should take note of. It is not a list of all open issues (see
[GitHub Issues](https://github.com/arangodb/arangodb/issues) for that).
```

Author: thanks - the problem with adding the link "https://github.com/arangodb/arangodb/issues" is that it is a partial list as other issues are in other repositories

> ArangoDB Customers can access the [_Technical & Security Alerts_](https://arangodb.atlassian.net/wiki/spaces/DEVSUP/pages/223903745) page after logging in the Support Portal.

Reviewer: I think we would have to write "logging in in the Support Portal" or "logging into the Support Portal", but they both sound a bit weird. Therefore, I suggest to change it from verb to noun.
Also see https://www.englishforums.com/English/LogInIntoIntoOnto/vzdxc/post.htm
```suggestion
ArangoDB Customers can access the
[_Technical & Security Alerts_](https://arangodb.atlassian.net/wiki/spaces/DEVSUP/pages/223903745)
page after login in the Support Portal.
```

| # | Issue      |
|---|------------|

> | 4 | **Date Added:** 2018-11-30 <br> **Component:** arangod <br> **Deployment Mode:** All <br> **Description:** Wrong suggestion printed in the log on how to optimize an OS setting, if followed, could cause ArangoDB to run into problems as the number of memory mappings will keep growing <br> **Affect Versions:** 3.3.0 to 3.3.19 <br> **Fixed in Versions:** 3.3.20 <br> **Reference:** [Link for ArangoDB Customers](https://arangodb.atlassian.net/plugins/servlet/servicedesk/customer/confluence/shim/spaces/DEVSUP/pages/228622337/Technical+Alert+%232%3A+Set+Linux+variable+overcommit_memory+to+0+or+1)  |

Reviewer: Redirects to KB page, so we can use shorter link can't we?

Should it be *ArangoDB Server* instead of  `arangod` as component?

```suggestion
| 4 | **Date Added:** 2018-11-30 <br> **Component:** arangod <br> **Deployment Mode:** All <br> **Description:** Wrong suggestion printed in the log on how to optimize an OS setting, if followed, could cause ArangoDB to run into problems as the number of memory mappings will keep growing <br> **Affect Versions:** 3.3.0 to 3.3.19 <br> **Fixed in Versions:** 3.3.20 <br> **Reference:** [Link for ArangoDB Customers](https://arangodb.atlassian.net/servicedesk/customer/kb/view/228622337)  |
```

Author: I would prefer we use arangod for the moment, as arangod can be also agency / coordinator etc

| 3 | **Date Added:** 2018-11-16 <br> **Component:** Security <br> **Deployment Mode:** All <br> **Description:** Unauthorized access to ArangoDB when using LDAP authentication <br> **Affect Versions:** 3.3.0 to 3.3.18 <br> **Fixed in Versions:** 3.3.19 <br> **Reference:** [Link for ArangoDB Customers](https://arangodb.atlassian.net/servicedesk/customer/kb/view/223903752)  |
| 2 | **Date Added:** 2018-11-03 <br> **Component:** Backup/Restore <br> **Deployment Mode:** All <br> **Description:** Users not included in the backup if _--server.authentication = true_ <br> **Affect Versions:** 3.3.0 to 3.3.13 <br> **Fixed in Versions:** 3.3.14 <br> **Reference:** [Link for ArangoDB Customers](https://arangodb.atlassian.net/servicedesk/customer/kb/view/226557953) |

> | 1 | **Date Added:** 2018-04-09 <br> **Component:** Storage <br> **Deployment Mode:** Single Instance <br> **Description:** Data corruption could happen under Linux <br> **Affect Versions:** 3.3.0 <br> **Fixed in Versions:** 3.3.1 <br> **Reference:** [Link for ArangoDB Customers](https://arangodb.atlassian.net/plugins/servlet/servicedesk/customer/confluence/shim/spaces/DEVSUP/pages/164069377/Important+Note+for+Users+running+ArangoDB+v.+3.2.4%2C+3.2.5%2C+3.2.6%2C+3.2.7%2C+3.2.8%2C+3.2.9+or+3.3.0+on+Linux)  |

Reviewer: Shorter KB link
```suggestion
| 1 | **Date Added:** 2018-04-09 <br> **Component:** Storage <br> **Deployment Mode:** Single Instance <br> **Description:** Data corruption could happen under Linux <br> **Affect Versions:** 3.3.0 <br> **Fixed in Versions:** 3.3.1 <br> **Reference:** [Link for ArangoDB Customers](https://arangodb.atlassian.net/servicedesk/customer/kb/view/164069377)  |
```
